packages/repository-tests/src/crud/relations/acceptance/repository.acceptance.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  AnyObject,
+  DefaultCrudRepository,
+  Entity,
+  model,
+  property,
+} from '@loopback/repository';
+import {juggler} from '@loopback/repository/src';
+import {expect, skipIf} from '@loopback/testlab';
+import {Suite} from 'mocha';
+import {
+  CrudFeatures,
+  CrudRepositoryCtor,
+  CrudTestContext,
+  DataSourceOptions,
+} from '../../..';
+import {
+  deleteAllModelsInDefaultDataSource,
+  withCrudCtx,
+} from '../../../helpers.repository-tests';
+import {Product} from '../fixtures/models';
+import {ProductRepository} from '../fixtures/repositories';
+
+export function hasOneRelationAcceptance(
+  dataSourceOptions: DataSourceOptions,
+  repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
+) {
+  // This test shows the recommended way how to use @loopback/repository
+  // together with existing connectors when building LoopBack applications
+  describe('Repository in Thinking in LoopBack', () => {
+    let repo: ProductRepository;
```

Comment:
I am little bit confused about this file. It's stored in `packages/repository-tests/src/crud/relations/acceptance/repository.acceptance.ts` but it does not seem to leverage any relations. What am I missing?

We already have other test files to verify the core Repository APIs, see e.g. [packages/repository-tests/src/crud/freeform-properties.suite.ts](https://github.com/strongloop/loopback-next/blob/6ad0bb59c71ea2356cb951da786bdbff246b47e7/packages/repository-tests/src/crud/freeform-properties.suite.ts) that's verifying non-strict (free-form) mode.